newm-server/src/main/kotlin/io/projectnewm/server/features/cloudinary/CloudinaryRoutes.kt

```diff
@@ -20,7 +22,7 @@ fun Routing.createCloudinaryRoutes() {
             with(call) {
                 val timestamp = Instant.now().epochSecond
                 val params = mutableMapOf<String, Any>().apply {
-                    putAll(receive<Map<String, String>>())
+                    receive<Map<String, JsonPrimitive>>().mapValuesTo(this) { it.value.value }
```

Comment:
nit: What do you think about naming the extension `primitive`? That way this would read `it.value.primitive`?

Reply:
I thought about that. But then it would be `JsonPrimitive.primitive`.  The value.value happens just because we are walking a list of Pair, which is unusual